src/features/header-flyout/styles/HeaderFlyout.scss

```diff
@@ -13,8 +14,8 @@
 
     .overlay {
         padding: 2 * $flyout-header-spacer $flyout-list-padding 0;
-        // TODO: decide which BDL color applies here
-        border: 1px solid $bdl-gray-10;
+
+        @include bdl-Overlay-container;
```

Comment:
Is the `padding` value overridden by this - do we still need it?

Reply:
`bdl-Overlay-container` does not contain padding so we do need it still

Reply:
@alexkrolick from some of our earlier discussions, i opted not to include any internal padding to these mixins, to avoid needing overrides in all use cases, like our button styles

Reply:
Ok - might make sense to stick to the convention of `@include` blocks coming before overrides/local values though, just in case.